src/com/cloudogu/ces/dogubuildlib/EcoSystem.groovy

```diff
@@ -213,6 +224,14 @@ class EcoSystem {
         }
     }
 
+    /**
+     * Installs the given dogu and performs an upgrade on it.
+     * Before installation, the dogu is purged to ensure a green slate.
```

Comment:
:+1: For doc comments.

But: Did you mean `a clean slate`?

Reply:
Oops, I knew something was off about that but I couldn't quite figure it out.